driver/weird_test.go

```diff
@@ -0,0 +1,111 @@
+package driver
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+)
+
+func check(err error, t *testing.T) {
```

Comment:
I can reproduce the failure, but using this function hides where it actually happens. I can recover the information by eg. inserting a panic here, but please use `t.Fail*` directly at the failure place and remove this function, thank you.